app/models/bookmark.rb

```diff
@@ -0,0 +1,4 @@
+class Bookmark < ApplicationRecord
+validates :title, presence: true, allow_blank: false
+validates :url, presence: true, allow_blank: false
```

Comment:
Please add 2 line space in each line.

What is the difference between `allow_blank: false` and `presence: true` ?

Reply:
I don't want to allow the creation of a bookmark  if this has not a title & url. So I used allow_blank: false here.
But as I read in some articles allow_blank only validates nil, presence validates nil as well as empty.  So I think I can have presence: true here. Changing the code accordingly.